_docs/LitmusVagrantTesting.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: page
+title: Testing Puppet Modules with Litmus and Vagrant
+---
+
+## Testing Puppet Modules with Litmus and Vagrant
+ 
+### The Setup 
+
+For this demo I'm gonna run through the entire process from start to finish, I hope that you find it useful.
+To start with the first thing that you need to do is clone down the module that you want to test, in this case we'll be using `puppetlabs-scheduled_task`. This can easily be accomplished with a simple command:
```

Comment:
```suggestion
To start, the first thing that you need to do is clone down the module that you want to test; in this case we'll be using `puppetlabs-scheduled_task`. This can easily be accomplished with a simple command:
```

Reply:
done